### PR TITLE
fix: moved to yarn classic

### PR DESCRIPTION
**Pull request by gentlementlegen**
Maybe I am too unskilled to understand how it works but I could not get yarn 4 to publish a package with credentials through a CLI. I kept having a `YN0033` error for authentication, which is not documented. Yarn berry under the hood is supposed to use `npm publish`, so it should have been working.

In the end I just reverted to Yarn classic which is supposed to work. If it still doesn't, it means the token itself is invalid.

It is on purpose that this PR target `main`, to trigger the publish and test. If it works I'll merge back to `develop`.

### CODE REVIEW

File: .github/workflows/publish-package.yml [hunk 1]
```diff
@@ -36,9 +36,10 @@ jobs:
...
           yarn build
           yarn pack
+        if: ${{ steps.release.outputs.release_created }}
 
       - name: Publish
-        run: yarn npm publish --access public
+        run: yarn publish --access public
         env:
-          NPM_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (0x4007) [inline .github/workflows/publish-package.yml:44]: Is this correct to change?

**Comment by 0x4007** [comment]
I have no experience with this.

File: .github/workflows/publish-package.yml (hunk 1 above)
Comment (gentlementlegen) [reply .github/workflows/publish-package.yml:44]: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry

**Review by Keyrxng — APPROVED** [review]
I don't see any issue with this, I was never against either version but I do think we should start using the `packageManager` field religiously. It's purpose is to prevent "works on my machine" issues and it's been a problem in CI a few times

> I am too unskilled to understand how it works

I feel the same, clearly V4 has perks but it's a bit of a nightmare to use vs V1

**Comment by gentlementlegen** [comment]
@Keyrxng Sure agreed, we can start putting the classic package version there moving forward. But yeah v4 has been painful to use so far it seems.

**Pull request merged**